tools/build_script_generator/scons/resources/build_target.py.in

```diff
@@ -14,7 +14,12 @@ from os.path import abspath, relpath
 
 def build_target(env, sources):
 	# Building application
-	program = env.Program(target=env["CONFIG_PROJECT_NAME"]+".elf", source=sources)
+	%% if target.family == "windows"
+	program_extension = ".exe"
+	%% else
+	program_extension = ".elf"
+	%% endif
+	program = env.Program(target=env["CONFIG_PROJECT_NAME"]+program_extension, source=sources)
```

Comment:
Do you want to move this into python? It'll be easier to reuse this way for CMake (once we add hosted support).
```py
env.substitutions["program_extension"] = ".exe" if target.family == "windows" else ".elf"
```
```suggestion
	program = env.Program(target=env["CONFIG_PROJECT_NAME"]+{{ program_extension }}, source=sources)
```

Reply:
:+1: updated